index.html

```diff
@@ -43,10 +21,8 @@
   <div class="width-wrap" >
     <header id="top">
          <nav>    
-            <label class="burger">
             <input type="checkbox" id="burger-toggle">
-            <label for="checkbox">checkbox</label>
-            </label>
```

Comment:
Incorrect HTML configuration.  The label and the checkbox need to be SIBLINGS, and the label needs to come AFTER the checkbox